provider/scaleway/scaleway.go

```
@@ -65,7 +65,13 @@ func NewScalewayProvider(ctx context.Context, domainFilter endpoint.DomainFilter
 			defaultPageSize = 1000
 		}
 	}
-	p, _ := scw.MustLoadConfig().GetActiveProfile()
+
+	p := &scw.Profile{}
+	c, err := scw.LoadConfig()
+	if err == nil {
```

good path on the right and error handling on the left, please.

---

No I mean:
```
if err != nil {
   // here is the right 
} else {
    // optional "good" path
}
// here is the left
```

Right now you ignore the error, which is not what we want.